lib/dsl/ast_builder.js

```diff
@@ -482,6 +482,8 @@ class JDLAstBuilderVisitor extends BaseJDLCSTVisitor {
       return stringImage.substring(1, stringImage.length - 1);
     } if (ctx.BOOLEAN) {
       return ctx.BOOLEAN[0].image === 'true';
+    } if (ctx.NAME_CAN_START_DIGIT) {
```

Comment:
Didn't notice what was previously there, but each if should start on its own line.

Reply:
It was a series of "else if" which you modified to plain ifs in this commit
https://github.com/jhipster/jhipster-core/commit/fe6614892f03cf02b9baf84a17546ea93140a605#diff-9fa3402fa25eaa022ea233eb8dbb86c9
Which is why the formatting is not valid...

But style X or style Y are irrelevant, the main issue is that formatting is not done **automatically.** Basically we(Humans) should not invest time in doing something a computer would do better and more consistently (prettier/standardjs).

Particularly as we all have day jobs and limited time to invest in OSS. 😭

Reply:
I'm always ok to add prettier. I thought it was already done here??

Reply:
I'm sorry just found the old ticket and realized I was being ignorant there, I think aging a bit more has made me a bit wiser :wink: 
https://github.com/jhipster/jhipster-core/issues/158

Reply:
np, its good to use this example use case to re-raise the question of standard consistent formatting tool. 👍